nautobot/utilities/testing/views.py

```diff
@@ -850,10 +850,6 @@ def test_bulk_edit_objects_without_permission(self):
                 "_apply": True,  # Form button
             }
 
-            # Test GET without permission
-            with disable_warnings("django.request"):
-                self.assertHttpStatus(self.client.get(self._get_url("bulk_edit")), 403)
```

Comment:
Remind me why we're removing these tests?

Reply:
Yeah so for a user to access the get method for bulk_delete and bulk_update in the UI, the only way it is possible is to type in the url for example, `circuits/circuits/delete` or `circuits/circuits/edit`, and all it does is returning the user to ListView. Furthermore, test_bulk_delete_objects_with_constrained_permissions(), test_bulk_delete_objects_with_permissions() do not test the get method at all.

Reply:
- [ ] Need to add error handling for key error: None